test/runner/requirements/integration.txt

```diff
@@ -7,3 +7,4 @@ passlib
 pexpect
 pycrypto
 pyyaml
+netaddr
```

Comment:
I'm revising my previous suggestion to add this to the requirements here. Instead, please use the pip module to install the requirement at the start of the test. That way we don't install test-specific dependencies when not running those tests.